### PR TITLE
[e2e] Destroy the created runtimeclass resources at the end of the test case.

### PR DESCRIPTION
**Pull request by tanjing2020**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Destroy the created runtimeclass resources at the end of the test case.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by aojea** [comment]
/lgtm
/retest
1 storage test failure unrelated

/assign @SergeyKanzhelev @BenTheElder

File: test/e2e/common/node/runtimeclass.go [hunk 1]
```diff
@@ -51,6 +51,7 @@ var _ = SIGDescribe("RuntimeClass", func() {
 		handler := f.Namespace.Name + "-handler"
 		rcName := createRuntimeClass(f, "unconfigured-handler", handler)
```
Comment (BenTheElder) [inline test/e2e/common/node/runtimeclass.go:52]: isn't it this call that actually makes the runtime class? why are we only deferring after creating the pod?

Comment (tanjing2020) [reply test/e2e/common/node/runtimeclass.go:52]: I have adjusted the position, please check again. Thanks.

Comment (tanjing2020) [reply test/e2e/common/node/runtimeclass.go:52]: I have adjusted the position, please check again. Thanks.

**Review by BenTheElder — APPROVED** [review]
/lgtm
/approve
thanks!

**Pull request merged**